app/scripts/views/mixins/password-mixin.js

```diff
@@ -124,6 +135,31 @@ define(function (require, exports, module) {
       }
     },
 
+    handleShowPasswordLabel (el) {
+      const $passwordEl = $(el);
+      if ($passwordEl.siblings(SHOW_PASSWORD_LABEL_SELECTOR).length === 0) {
+        // if no label exists, create one
+        this.createShowPasswordLabel($passwordEl);
+      }
+      // update the visibility of the label
+      this.updateShowPasswordLabelVisibility($passwordEl);
+    },
+
+    createShowPasswordLabel ($passwordEl) {
+      const $showPasswordLabel = $('<label for="show-password">' + SHOW_MESSAGE + '</label>');
```

Comment:
See the [permission partial](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/templates/partial/permission.mustache) and how it's used in the [permissions view](https://github.com/mozilla/fxa-content-server/blob/05db4d25291f4de81ad8e38ea7304c5bcf4dca79/app/scripts/views/permissions.js#L223) - no new View needed, just render the partial and write the resulting HTML into the DOM afterwards. Use of the map there to get the data to pass to the template is a bit strange, but once you understand what's going on, makes sense.